### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by amon-cofie**
### Changes: 

- Implemented the sneak peek wireframes
- Added views for all the URLs in the sneak peek
- Added routes to all the pages in the sneak peek

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,30 @@
...
+    </div>
+</div>
+<div>
+    <p>Bio</p>
+    <p>
+        <%=@user.bio%>
+    </p>
+</div>
+<div>
+    <% @posts.each do |post| %>
```
Comment (Ismailco) [inline app/views/users/show.html.erb]: - [ ] Kindly make sure to use `recent_posts` method to render only the last three posts in the user `show` page

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+        <div>
+            <p><%=@user.name%></p>
+            <p><%=@user.posts_counter%></p>
+        </div>
+    </div>
+
+    <div>
+        <%@posts.each do |post|%>
+        
+            <%= render partial: "post_card", locals: {post: post, comments: @comments, user: @user}%>
```
Comment (Ismailco) [inline app/views/posts/index.html.erb]: - [ ] Same here kindly make sure to use `recent_comments` method to render only the last 5 comments in the `posts` index page as required

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @amon-cofie, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [ ] Kindly make sure your pages look like the provided wireframes

||||
|-|-|-|
|Screenshot_20230203_165918|Screenshot_20230203_165923|Screenshot_20230203_170141|
|Screenshot_20230203_165938|Screenshot_20230203_165943|Screenshot_20230203_170154|
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+        <%= render partial: "post_card", locals: {post: post, comments: @comments, user: @user}%>
+
+    <%end%>
+</div>
+    
+<div class="posts_index_pagination_button">    
+    <button>
+        Pagination        
+    </button>
+</div>
```
Comment (muneebulrehman) [inline app/views/posts/index.html.erb:25]: As pointed out by the previous reviewer, the post index page should display the posts with recent five comments. However, in your application, though it displays the comments counter correctly, it doesn't display the recent five comments. Kindly fix this issue. 
| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @amon-cofie , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors
- Descriptive PR
- Good UI

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (amon-cofie) [reply app/views/posts/index.html.erb:25]: Thank you but I think I would need more context to this as I implemented this and upon checking just now, works just fine: 

[GIF](https://www.webmobilefirst.com/en/screencasts/-h4afY7FhN/)

**Review by mugishasam123 — APPROVED** [review]
Hi @amon-cofie  :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors :+1: 
- Descriptive PR summary, keep rocking :100: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**